community/cypher/cypher-compiler-3.0/src/test/scala/org/neo4j/cypher/internal/compiler/v3_0/helpers/ScalaCompatibilityTest.scala

```diff
@@ -103,6 +103,6 @@ class ScalaCompatibilityTest extends CypherFunSuite {
   test("should convert traversable to Iterable") {
     val it = Stream[Any](1, 2, 3)
 
-    it shouldBe a [Iterable[Any]]
+    it shouldBe a [Iterable[_]]
```

Comment:
If you're fixing this, might as well correct the language and write `it shouldBe an [Iterable[_]]`